### ISSUE TITLE
Deploy Proofing Agent App to LHDI DEV Region - CI

### ISSUE THREAD

**Issue by porta-antiporta** [department-of-veterans-affairs/va.gov-team#68453]
# Context

We have a repo for our proofing app work which now has the code for Hello World, let's deploy it to QA

# Acceptance Criteria
- [ ] Build pipeline established in repo to deploy to DEV
- [ ] Proofing agent app deployed to DEV
- [ ] Design / Product can access Proofing Agent app in DEV

Update: We are splitting this ticket into a CI ticket and differentiating it from CD tasks which will be captured in [insert ticket]. See comments below for more info.

# Updated AC for CI

- [ ] Checkout build
- [ ] Run automated processes: unit tests, CodeQL, linting, formatting
- [ ] Push to VA GH Container Registry

**Comment by rdvyas** [comment]
Worked with LHDI team contact Tyler, we were able to resolve the issues with Lightkeeper CLI that was preventing us from working with the LHDI infrastructure. 

We have resolved these blockers (issue was related to creating the k8s config on CAG environment, we were provided with a one of script to get around our issue. This has been logged as a ticket with LHDI team to prevent future issues with teams on CAG)

Next step is to perform the manual deployment of IPP App to Dev region.

**Comment by rdvyas** [comment]
- Working to complete deployment to DEV region prior to meeting with LHDI team. 
-

**Comment by rdvyas** [comment]
@skirkman16 - After working on this ticket, I think it would make make sense to decompose this ticket into smaller tickets to better track and move the work items through the queue. 

The goal of this ticket was to track the SecDevOps efforts for the CI/CD pipeline targeting the DEV environment. We could then use this same pipeline to promote to higher environments. 

We should split this into two tickets, one for Ci, the other for CD portion of the process. 

Currently we have accomplished all of the tasks noted in Green in the diagram below:
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK0dUQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--026d37655f0f31c9bd48f2583f44f57a16d512f3/image.png" alt="image.png" />

Namely we have completed the following CI tasks:

1. Checkout build
2. Run automated processes: unit tests, CodeQL, linting, formatting
3. Push to VA GH Container Registry

The results of this is that we have a Docker image pushed to the VA GHCR which we can now use for the deployment activities:

[IPP Docker Package](https://github.com/orgs/department-of-veterans-affairs/packages/container/package/va-in-person-identity-proofing%2Fva-in-person-identity-proofing)

tl;dr - would recommend splitting the pipeline work into CI and CD tickets, the activities above could be captured in CI portion. Create a new ticket for CD.
